package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "testts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
Please remove this file.

----

Joke for @moncapitaine: Actually, I think it's good if there is one package.json in the root directory. 😉